stepfunctions/src/test/java/com/github/awsjavakit/stepfunctions/lambda/StepFunctionHandlerTest.java

```diff
@@ -29,11 +32,26 @@ class StepFunctionHandlerTest {
   public static final String MISSING_FIELD_NAME = "someString";
   private ByteArrayOutputStream outputStream;
 
+  public static Stream<BaseType> polymorphicProvider() {
+    return Stream.of(new SubTypeA(), new SubTypeB());
```

Comment:
The classes and interface could have been inner classes for encapsulation purposes (and easier to purge), but not important